src/tests/const.test.js

```diff
@@ -156,4 +156,35 @@ describe('const/default with forced values', () => {
     // Expectation: To fail with error "The only accepted value is 10."
     expect(handleValidation({ ten_only: null }).formErrors).toBeUndefined();
   });
+
+  it('Should clear the default from number when removed from condition', () => {
```

Comment:
todo: Can you check the tests for `schemaWithConditionalReadOnlyProperty`? Now we should be able to remove the redundant conditional `readOnly:false` (because it's the default state) and still pass.